### PR TITLE
Refactor Wallet Activities Query

### PR DESCRIPTION
**Pull request by alchemistgo87**
- Add limit, offset, event types as variables to wallet activities query
- Remove cancelled_listing event from wallet activities query

### CODE REVIEW

**Review by kespinola — APPROVED** [review]
nice. way to get the pagination and event_types in 👍

**Review by kespinola — CHANGES_REQUESTED** [review]
@alchemistgo87 what is using this query right now? I'm think what UI will break from this change.

**Comment by alchemistgo87** [comment]
> @alchemistgo87 what is using this query right now? I'm think what UI will break from this change.

I suppose right now this can only break the current profile activity page at holaplex.com

File: crates/core/src/db/queries/wallet.rs [hunk 1]
```diff
@@ -10,62 +15,75 @@ use crate::{
...
+) -> Result<Vec<WalletActivity>> {
+    let query = diesel::sql_query(ACTIVITES_QUERY)
         .bind(address)
+        .bind(event_types)
+        .bind(limit)
+        .bind(offset)
         .load(conn)
-        .context("Failed to load wallet(s) activities")
+        .context("Failed to load wallet(s) activities");
+    println!("Query Result: {:?}", query);
```
Comment (kespinola) [inline crates/core/src/db/queries/wallet.rs:84]: This println should be dropped.

**Comment by kespinola** [comment]
https://github.com/holaplex/holaplex/pull/701

Holaplex updated to work with pagination.

**Review by kespinola — APPROVED** [review]

**Pull request merged**